src/main/java/Calculator.java

```diff
@@ -0,0 +1,20 @@
+public class Calculator {
+
+    public double add(double a1, double a2){
+        return a1 + a2;
+    }
+
+    public double subtract(double a1, double a2){
+        return a1 - a2;
+    }
+
+    public double multiply(double a1, double a2){
```

Comment:
Please time from time use CTRL+ALT+L reformat code and CTRL + ALT + O optimize imports shortcut.
Especially before commiting or show to someone your code.

Code should be formatted, Idea could do it for you in one shortcut.

`(double a1, double a2){  -> (double a1, double a2) { `